src/PostgREST/Query/Statements.hs

```diff
@@ -151,8 +151,10 @@ callProcStatement returnsScalar returnsSingle callProcQuery selectQuery countQue
     (countCTEF, countResultF) = countF countQuery countTotal
 
     bodyF
+     | asXml              = asXmlSingleF returnsScalar
```

Comment:
@fjf2002 There's no `asXmlSingleF` defined, that's why all compilation/tests are failing.

Reply:
Thanks. I am not used to `git rebase`, took me quite a while and - well, this code line was re-introduced.